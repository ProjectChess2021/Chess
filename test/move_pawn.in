game human human
move a2 a3
move a7 a6
move d2 d4
move f7 f5
move a3 a4
move a6 a5