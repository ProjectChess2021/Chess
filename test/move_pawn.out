8 rnbqkbnr      MoveHistory
7 pppppppp
6  _ _ _ _
5 _ _ _ _
4  _ _ _ _
3 _ _ _ _
2 PPPPPPPP
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr      MoveHistory
7 pppppppp      White: pawn a2 -> a3
6  _ _ _ _
5 _ _ _ _
4  _ _ _ _
3 P _ _ _
2  PPPPPPP
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr      MoveHistory
7 _ppppppp      White: pawn a2 -> a3
6 p_ _ _ _      Black: pawn a7 -> a6
5 _ _ _ _
4  _ _ _ _
3 P _ _ _
2  PPPPPPP
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr      MoveHistory
7 _ppppppp      White: pawn a2 -> a3
6 p_ _ _ _      Black: pawn a7 -> a6
5 _ _ _ _       White: pawn d2 -> d4
4  _ P _ _
3 P _ _ _
2  PP_PPPP
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr      MoveHistory
7 _ppppp_p      White: pawn a2 -> a3
6 p_ _ _ _      Black: pawn a7 -> a6
5 _ _ _p_       White: pawn d2 -> d4
4  _ P _ _      Black: pawn f7 -> f5
3 P _ _ _
2  PP_PPPP
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr      MoveHistory
7 _pppp pp      White:  pawn a2 -> a3
6 p_ _ _ _      Black:  pawn a7 -> a6
5 _ _ _p_       White:  pawn d2 -> d4
4 P_ P _ _      Black:  pawn f7 -> f5
3 _ _ _ _       White:  pawn a3 -> a4
2  PP_PPPP
1 RNBQKBNR

  abcdefgh
8 rnbqkbnr                                                              MoveHistory (Newest at top)
7 _pppp pp                                                              Black: a6 -> a5
6  _ _ _ _                                                              White: a3 -> a4
5 p _ _p_                                                               Black: f7 -> f5
4 P_ P _ _                                                              White: d2 -> d4
3 _ _ _ _                                                               Black: a7 -> a6
2  PP_PPPP                                                              
1 RNBQKBNR                                                              
  abcdefgh
